src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/store/ElasticSearchAnalysisGraph.java

```diff
@@ -124,22 +130,31 @@ public void construct() {
     highCpuRca.addTag(TAG_LOCUS, LOCUS_DATA_MASTER_NODE);
     highCpuRca.addAllUpstreams(Collections.singletonList(cpuUtilizationGroupByOperation));
 
-    Rca<ResourceFlowUnit> hotJVMNodeRca = new HotNodeRca(12, highHeapUsageOldGenRca,
-            highHeapUsageYoungGenRca, highCpuRca);
-    hotJVMNodeRca.addTag(TAG_LOCUS, LOCUS_DATA_MASTER_NODE);
-    hotJVMNodeRca.addAllUpstreams(
-            Arrays.asList(highHeapUsageOldGenRca, highHeapUsageYoungGenRca, highCpuRca));
+    Rca<ResourceFlowUnit> queueRejectionRca = new QueueRejectionRca(12, threadPool_RejectedReqs);
+    queueRejectionRca.addTag(TAG_LOCUS, LOCUS_DATA_MASTER_NODE);
+    queueRejectionRca.addAllUpstreams(Collections.singletonList(threadPool_RejectedReqs));
+
+    Rca<ResourceFlowUnit> hotNodeRca = new HotNodeRca(12, highHeapUsageOldGenRca,
```

Comment:
A bunch of the RCAs take `12` as the constant. Can we create a variable with appropriate name and use it for readability ?

Reply:
done